fv3core/.jenkins/actions/run_parallel_regression_tests.sh

```diff
@@ -16,5 +16,9 @@ export TEST_ARGS="${TEST_ARGS} --compute_grid"
 if [ ${python_env} == "virtualenv" ]; then
     CONTAINER_CMD="" make savepoint_tests_mpi
 else
-    make savepoint_tests_mpi
+    if [[ ${FV3_DACEMODE} == "True" ]]; then
+        RUN_FLAGS="--rm -e FV3_DACEMODE=True" make savepoint_tests_mpi
```

Comment:
Can you put this logic in the Makefile instead, so that it happens automatically when we test as a user? I'd expect to be able to run `FV3_DACEMODE=True TEST_ARGS="--backend=\"gtc:gt:dace\" make savepoint_tests` (modulo using the correct backend name, if that's not it). If you want to automatically set the backend when `FV3_DACEMODE=True` to remove the TEST_ARGS bit, I wouldn't complain (but be on the lookout for FV3_DACEMODE=True being set when running other-backend tests on jenkins).

Reply:
Moved to Makefile.